editor/src/com/talosvfx/talos/editor/nodes/EmptyWindow.java

```diff
@@ -218,7 +219,7 @@ public boolean keyTyped(InputEvent event, char character) {
 
     public void draw(Batch batch, float parentAlpha) {
         Stage stage = this.getStage();
-        if (stage != null && stage.getKeyboardFocus() == null) {
+        if (stage != null && stage.getKeyboardFocus() == null && SharedResources.stage.getKeyboardFocus() == null) {
```

Comment:
This is most likely not the solution. We shouldn't be doing any keyfocus stuff in draw method. Its ok for now, but real solution is to take all of this out, and make sure focus is only taken when we know the user wants to take it